examples/contour/01-crds.yaml

```diff
@@ -2,6 +2,8 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
gross, is there a way to remove this?

Reply:
I don't think so, but I'll check.

Reply:
Looks like this is the desired behaviour, we've pulled it in when upgrading controller-gen, it seems. https://github.com/kubernetes-sigs/controller-tools/pull/338 and https://github.com/kubernetes-sigs/controller-tools/issues/348 have more detail.

Reply:
I'm confused about this as well. The only tooling we'd need to consume these CRDs is possibly to create the generated `DeepCopy` types.

Reply:
In order to actually generate these CRDs, we use controller-gen, specifically:
```sh
go run sigs.k8s.io/controller-tools/cmd/controller-gen \
  crd paths=./apis/... output:dir=$TEMPDIR
```
this is part of `controller-tools`, which had to be updated from 0.2.2 to 0.2.4 as part of this change. This behaviour can't be disabled at the moment, sadly. https://github.com/kubernetes-sigs/controller-tools/pull/338 was merged before 0.2.4 was cut for controller-tools, so it got pulled in for us.